Simlar/src/SMLRLinphoneHandler.m

```diff
@@ -203,6 +210,28 @@ - (void)initLibLinphone
     [SMLRLinphoneHandler enableExternalSpeaker:NO];
 }
 
+- (void)terminatePossibleIncomingCall
+{
+    if (_linphoneCore == NULL) {
+        return;
+    }
+
+    LinphoneCall *const call = [self getCurrentCall];
+    if (call == NULL) {
+        SMLRLogI(@"terminatePossibleIncomingCall no current call");
+        return;
+    }
+
+    const LinphoneCallState state = linphone_call_get_state(call);
```

Comment:
[self hasIncomingCall] would do almost the same, except the state check for LinphoneCallIncomingEarlyMedia. I would add this check there and reuse it here.